lib/resources/ssh_conf.rb

```diff
@@ -63,7 +63,7 @@ def read_content
       end
 
       @content = file.content
-      if @content.empty? && !file.empty?
+      if @content.nil? || (@content.empty? && File.size?(@conf_path))
```

Comment:
We can't use the Ruby `File` class because this will always operate locally on the machine executing InSpec. If the user is scanning a remote target with `--target`, this call will still happen on the local machine. That's why the call to `file` was being done (lowercase `file` referring to the variable storing the `inspec.file` resource that will work both remotely and locally).

Reply:
There is no inspec.file.empty? currently. Should we create one that does a size check accordingly?

Reply:
Since we basically just hand over logic on `file`s to Train, it should probably be implemented there. Can we do `file.size == 0` for now and address that in a future PR?

Reply:
sounds good to me